File: cubular-client/js/messageType.js
function MessageType() {
};

MessageType.SpawnRequested = 0;
MessageType.SpawnAccepted = 1;
MessageType.MovementRequest = 2;
MessageType.MovementAccepted = 3;
MessageType.MovementRejected = 4;
MessageType.CubeInfo = 5;
MessageType.CubeMovement = 6;
MessageType.CubeDestroyed = 7;

module.exports = MessageType;

File: cubular-server/CubularServer/Network/Messages/MessageType.cs
namespace CubularServer.Network.Messages
{
    public enum MessageType
    {
        SpawnRequested,
        SpawnAccepted,
        MovementRequest,
        MovementAccepted,
        MovementRejected,
        CubeInfo,
        CubeMovement,
        CubeDestroyed,
    }
}
